template/src/api/index.js

```diff
@@ -20,11 +21,12 @@ export default function indexEntrypointFile({asyncapi, params}) {
     const ${capitalizedProtocol}Adapter = require('hermesjs-${protocol}');
     `;
 
-    const channelHandlerImports = Object.entries(asyncapi.channels()).map(([channelName, channel]) => {
+    const channelHandlerImports = asyncapi.channels().all().map(channel => {
+        const channelName = channel.address();
```

Comment:
since in v3 address is no longer also a channel name, I suggest you change it to `channel.id()`